tests/suit/openstack_test.go

```diff
@@ -19,6 +19,63 @@ const (
 var _ = Describe("[level:component]Migration tests for OpenStack provider", func() {
 	f := framework.NewFramework("migration-func-test")
 
+	It("[test] should create OSP provider using https/ssl with CA", func() {
+		namespace := f.Namespace.Name
+		err := f.Clients.OpenStackClient.SetupClient("cirros-volume", "net-int", "nfs")
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Load Source VM Details from OpenStack")
+		packstackCA, err := f.Clients.OpenStackClient.LoadCA(f, packstackNameSpace, "packstack")
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Create Secret from Definition")
+		s, err := utils.CreateSecretFromDefinition(f.K8sClient, utils.NewSecretDefinition(nil,
+			map[string]string{
+				"username":    "admin",
+				"password":    "12e2f14739194a6c",
+				"domainName":  "default",
+				"projectName": "admin",
+				"regionName":  "RegionOne",
+				"cacert":      packstackCA,
+			}, nil, namespace, "os-test-secret"))
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Create osp provider")
+		pr := utils.NewProvider(openstackProviderName, forkliftv1.OpenStack, namespace, map[string]string{},
+			"https://packstack.konveyor-forklift:5001/v3", s)
+		err = utils.CreateProviderFromDefinition(f.CrClient, pr)
+		Expect(err).ToNot(HaveOccurred())
+		err = utils.WaitForProviderReadyWithTimeout(f.CrClient, namespace, openstackProviderName, 30*time.Second)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	It("[test] should create OSP provider using https/ssl insecure", func() {
+		namespace := f.Namespace.Name
+		err := f.Clients.OpenStackClient.SetupClient("cirros-volume", "net-int", "nfs")
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Create Secret from Definition")
+		s, err := utils.CreateSecretFromDefinition(f.K8sClient, utils.NewSecretDefinition(nil,
+			map[string]string{
+				"username":           "admin",
+				"password":           "12e2f14739194a6c",
+				"domainName":         "default",
+				"projectName":        "admin",
+				"regionName":         "RegionOne",
+				"insecureSkipVerify": "true",
+				"cacert":             "",
+			}, nil, namespace, "os-test-secret"))
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Create osp provider")
+		pr := utils.NewProvider(openstackProviderName, forkliftv1.OpenStack, namespace, map[string]string{},
```

Comment:
try a different provider name? maybe the secured should have suffix `secured` and here `insecured`.

Reply:
okay, i've made the change